src/utilities/scroll-lock-manager/scroll-lock-manager.ts

```diff
@@ -2,9 +2,7 @@ import {isServer} from '../target';
 
 export const SCROLL_LOCKING_ATTRIBUTE = 'data-lock-scrolling';
 
-export const SCROLL_LOCKING_WRAPPER_ATTRIBUTE = 'data-lock-scrolling-wrapper';
-
-export const SCROLL_LOCKING_CUSTOM_PROPERTY = '--scroll-lock-body-padding';
```

Comment:
are you sure we can remove this?

Reply:
I checked for usages and there were none outside the current file, I then removed the `export` and it complained that it was never used inside this file